docs/standard/generics.md

```diff
@@ -9,15 +9,15 @@ ms.assetid: a315b111-8e48-446c-ab19-acb6405894a7
 
-First introduced to the .NET Framework 2.0, generics involved changes to both the C# language and the Common Language Runtime (CLR). **Generics** are essentially a "code template" that allows developers to define [type-safe](https://msdn.microsoft.com/library/hbzz1a9a.aspx) data structures without committing to an actual data type. For example, `List<T>` is a [Generic Collection](xref:System.Collections.Generic) that can be declared and used with any type: `List<int>`, `List<string>`, `List<Person>`, etc.
+First introduced to the .NET Framework 2.0, generics involved changes to both the C# language and the Common Language Runtime (CLR). **Generics** are essentially a "code template" that allows developers to define [type-safe](https://msdn.microsoft.com/library/hbzz1a9a.aspx) data structures without committing to an actual data type. For example, `List<T>` is a [Generic Collection](xref:System.Collections.Generic) that can be declared and used with any type, such as `List<int>`, `List<string>`, or `List<Person>`.
 
-So, what’s the point? Why are generics useful? In order to understand this, we need to take a look at a specific class before and after adding generics. Let’s look at the `ArrayList`. In C# 1.0, the `ArrayList` elements were of type `object`. This meant that any element that was added was silently converted into an `object`; same thing happens on reading the elements from the list (this process is known as [boxing](../../docs/csharp/programming-guide/types/boxing-and-unboxing.md) and unboxing respectively). Boxing and unboxing have an impact of performance. More than that, however, there is no way to tell at compile time what is the actual type of the data in the list. This makes for some fragile code. Generics solve this problem by providing additional information the type of data each instance of list will contain. Put simply, you can only add integers to `List<int>` and only add Persons to `List<Person>`, etc.
+So, what’s the point? Why are generics useful? To understand this, let's take a look at a specific class before and after adding generics: <xref:System.Collections.ArrayList>. In C# 1.0, the `ArrayList` elements were of type [object](xref:System.Object). This meant that any element that was added was silently converted into an `object`. Same thing happens on reading the elements from the list (this process is known as [boxing](../../docs/csharp/programming-guide/types/boxing-and-unboxing.md) and unboxing respectively). Boxing and unboxing have an impact of performance. More than that, however, there's no way to tell at compile time what is the actual type of the data in the list. This makes for some fragile code. Generics solve this problem by providing additional information the type of data each instance of list will contain. For example, you can only add integers to `List<int>` and only add Persons to `List<Person>`.
```

Comment:
Again unrelated to your PR:
- object --> Object (.NET Framework type, instead of C# type)
- Same thing happens... --> much too informal for me.
- boxing and unboxing, respectively --> it's unclear which is which, since in the context of the sentence both boxing and unboxing apply to reading the elements from the list.
- to tell at compile time what is the actual type of the data in the list --> to determine the type of data in the list at compile time
- by providing additional information the type of data --> by defining the type of data